src/apps/chifra/pkg/pinlib/manifest/range.go

```diff
@@ -0,0 +1,51 @@
+package manifest
+
+import (
+	"strconv"
+	"strings"
+)
+
+type ManifestRange [2]uint64
+
+func (mr *ManifestRange) UnmarshalJSON(data []byte) error {
+	unquoted, err := strconv.Unquote(string(data))
+
+	if err != nil {
+		return err
+	}
+
+	newManifestRange, err := ManifestRangeFromString(unquoted)
+
+	if err != nil {
+		return err
+	}
+
+	*mr = newManifestRange
+
+	return nil
+}
+
+func ManifestRangeFromString(source string) (ManifestRange, error) {
+	mr := [2]uint64{0, 0}
+
+	if len(source) == 0 {
+		return mr, nil
+	}
+
+	parts := strings.Split(source, "-")
+
+	start, err := strconv.ParseUint(parts[0], 10, 64)
+	if err != nil {
+		return mr, err
+	}
+
+	end, err := strconv.ParseUint(parts[1], 10, 64)
```

Comment:
Would it be a valid manifest range then?
The function will report an error, because my understanding is that, as opposed to block ranges, manifest range is always `from-to`